lib/iex/lib/iex/helpers.ex

```diff
@@ -623,6 +624,21 @@ defmodule IEx.Helpers do
   defp history, do: Process.get(:iex_history)
 
   @doc """
+  Creates a PID from `string`.
+
+  ## Examples
+
+      iex> pid("0.21.32")
+      #PID<0.21.32>
+      iex> pid("0.64.2048")
+      #PID<0.64.2048>
+
+  """
+  def pid(string) when is_binary(string) do
+    "<#{string}>" |> String.to_char_list() |> :erlang.list_to_pid()
```

Comment:
We can interpolate string into char list directly: `:erlang.list_to_pid('<#{string}>')`.